archive/schema.go

```diff
@@ -6,20 +6,30 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sort"
 
 	"github.com/brimsec/zq/pkg/fs"
+	"github.com/brimsec/zq/pkg/nano"
 )
 
-const configName = "zar.json"
+const metaDataFilename = "zar.json"
```

Comment:
@alfred-landrum: Metadata is one word, not two, so this should be metadataFilename.